app/src/main/java/com/mapbox/maps/testapp/examples/java/RuntimeStylingJavaActivity.java

```diff
@@ -0,0 +1,423 @@
+            "    },\n" +
+            "    \"geometry\": {\n" +
+            "      \"type\": \"Point\",\n" +
+            "      \"coordinates\": [\n" +
+            "-27.861328125,\n" +
+            "3.337953961416485\n" +
+            "      ]\n" +
+            "    }\n" +
+            "  },\n" +
+            "  {\n" +
+            "    \"type\": \"Feature\",\n" +
+            "    \"properties\": {\n" +
+            "      \"count\": 1\n" +
+            "    },\n" +
+            "    \"geometry\": {\n" +
+            "      \"type\": \"Point\",\n" +
+            "      \"coordinates\": [\n" +
+            "-27.773437499999996,\n" +
+            "-17.644022027872712\n" +
+            "      ]\n" +
+            "    }\n" +
+            "  }\n" +
+            "]\n" +
+            "}";
+    private static final String FILL_FEATURE_COLLECTION = "{\n" +
+            "\"type\": \"FeatureCollection\",\n" +
+            "\"features\": [\n" +
+            "  {\n" +
+            "    \"type\": \"Feature\",\n" +
+            "    \"properties\": {},\n" +
+            "    \"geometry\": {\n" +
+            "      \"type\": \"Polygon\",\n" +
+            "      \"coordinates\": [\n" +
+            "        [\n" +
+            "          [\n" +
+            "            -366.85546875,\n" +
+            "            18.145851771694467\n" +
+            "          ],\n" +
+            "          [\n" +
+            "            -373.27148437499994,\n" +
+            "            12.726084296948196\n" +
+            "          ],\n" +
+            "          [\n" +
+            "            -364.39453125,\n" +
+            "            6.577303118123887\n" +
+            "          ],\n" +
+            "          [\n" +
+            "            -366.85546875,\n" +
+            "            18.145851771694467\n" +
+            "          ]\n" +
+            "        ]\n" +
+            "      ]\n" +
+            "    }\n" +
+            "  }\n" +
+            "]\n" +
+            "      }";
+    private MapboxMap mapboxMap;
+    private MapView mapView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_simple_map);
+        mapView = findViewById(R.id.mapView);
+        mapboxMap = mapView.getMapboxMap();
+        mapboxMap.loadStyleUri(Style.MAPBOX_STREETS, this::setUpStyle);
+    }
+
+    private void setUpStyle(Style style) {
+        addImage(style);
+        addSymbolSource(style);
+        addSymbolLayer(style);
+
+        addFillSource(style);
+        setFillLayer(style);
+        addFillExtrusionLayer(style);
+        addFillExtrusionLight(style);
+
+        addImageSource(style);
+        addRasterLayer(style);
+
+        addLayerWithoutStyleExtension(style);
+
+        VectorSource source = (VectorSource) SourceUtils.getSource(style, "composite");
+        Logger.e(TAG, "getSource: " + source);
```

Comment:
Guess should be either removed or changed from error to info

Reply:
I'd like to keep these logs to help us debug, but yeah can change to info

Reply:
I think it's already working so there's nothing to debug while leaving those pollutes code a bit
but up to you